api/types/types_drivers_executor.go

```diff
@@ -132,6 +158,7 @@ type ProvidesStorageExecutorCLI interface {
 // the driver implementations of the StorageExecutor interface.
 type StorageExecutorCLI interface {
 	StorageExecutorFunctions
+	StorageExecutorWithSupported
```

Comment:
And crap, I just realized I could use `StorageExecutorWithSupported` as the base of `StorageExecutorCLI` instead with the former using `StorageExecutorFunctions` as its base.

Dammit. Thanks @codenrhoden. I'll update the code. Good catch.